src/lib/chat/matrix-client.ts

```diff
@@ -341,6 +342,16 @@ export class MatrixClient implements IChatClient {
       name = roomNameEvent.getContent().name;
     }
 
+    let lastMessageEvent = null;
+    const timelineEvents = room.getLiveTimeline().getEvents();
+    for (let i = timelineEvents.length - 1; i >= 0; i--) {
+      if (timelineEvents[i].getType() === EventType.RoomMessage) {
+        lastMessageEvent = timelineEvents[i];
+        break;
+      }
+    }
+    const lastMessage = this.mapMatrixEventToMessage(lastMessageEvent);
```

Comment:
Since we have to filter through them all anyway....is there any reason not to just populate the messages we have in the `messages` property of the conversation? Then we don't need to set the lastMessage property at all.